app/templates/_brief_attributes.html

```diff
@@ -0,0 +1,34 @@
+{% import "toolkit/summary-table.html" as summary %}
+{% call(item) summary.list_table(
+  [
+    {"label": "Published", "value": brief.publishedAt|dateformat}
+  ],
+  caption="Important dates",
+  field_headings=[
+    "Brief attribute name",
```

Comment:
We shouldn't call these `Brief` in anything that reaches end users.  So the field headings here and further down this template should probably be `Opportunity...` instead.